planning/Development-strategy.md

```diff
@@ -27,6 +27,20 @@
   
 - "header-right button" border, font-weight, outline, background, color, padding, border-radius, cursor
 
+## `for: Main Image`
+
+### `type: html` (Main Image)
+
+- section with class called "container_header"
```

Comment:
just for your information in case you missed `it,`
"container_header" class is already exists in header part 
and i saw at 40.line "container img"
could you confirm that they are correct? thank you.

Reply:
I changed thank you